src/Components/CurrWeatherCard/CurrWeatherCard.jsx

```diff
@@ -13,13 +13,19 @@ export default function CurrWeatherCard({ locationData, currWeather }) {
     let year = selectedDate.toLocaleString('en-GB', {
       year: 'numeric'
     });
+
+    return `${date} ${year}`;
+  }
+
+  function formatTime() {
```

Comment:
formatDate and formatTime functions can be moved outside of the component body

Reply:
done